# bioinformatics-tools
Collection of bioinformatic tools

# CRISPR Target Deep Sequencing
- [CrispRVariants](http://www.bioconductor.org/packages/release/bioc/html/CrispRVariants.html) - [R] - CrispRVariants provides tools for analysing the results of a CRISPR-Cas9 mutagenesis sequencing experiment, or other sequencing experiments where variants within a given region are of interest. These tools allow users to localize variant allele combinations with respect to any genomic location (e.g. the Cas9 cut site), plot allele combinations and calculate mutation rates with flexible filtering of unrelated variants.
- [CRISP-ID](http://crispid.gbiomed.kuleuven.be) - [webtools] - CRISP-ID is a web application that allows the detection of the exact indel size and location of a CRISPR-Cas9 targeted region, based on direct Sanger sequencing. 
- [CRISPResso2](https://github.com/pinellolab/CRISPResso2) - [webtools, commandline] - Analysis of deep sequencing data for rapid and intuitive interpretation of genome editing experiments. 1-aligns sequencing reads to a reference sequence; 2-quantifies insertions, mutations and deletions to determine whether a read is modified or unmodified by genome editing; 3-summarizes editing results in intuitive plots and datasets.
- [Microhomology-Predictor](http://www.rgenome.net/mich-calculator/) - [webtools] - Calculating microhomology-associated scores for all engineered nucleases [ZFNs, TALENs, RGENs (Cas9 RNA-guided endonucleases), etc.]
